docs/docs/data-structures/tree/reading-and-editing.mdx

```diff
@@ -52,6 +52,10 @@ We show how to add this note to an array of notes in the tree in [Array node API
 ### Editing Object Properties
 
 To update the property on an object node, you assign a new node or value to it with the assignment operator (`=`).
+See [here](https://github.com/microsoft/FluidFramework/blob/main/packages/dds/tree/docs/user-facing/object-merge-semantics.md) for details on the merge semantics of these edits.
+
+Note that if the new value is a node (as opposed to a primitive), then its [status](./nodes.mdx#treestatus) must be `TreeStatus.New`,
```

Comment:
Is this important enough to list before the examples? I think allowing readers to digest the code first, and the warning second, might make this less of a rabbit hole.

Reply:
+1, but this section also requires some deduping with the quick start page so I feel like it's prob fine to make these changes as reminders of info we need to include and rearrange later

Reply:
Moved it down